controller/internal/controller/cluster_controller.go

```diff
@@ -127,6 +128,11 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req reconcile.Request
 
 	// Sync resource with Console API.
 	apiCluster, err := r.sync(ctx, cluster, providerId, projectId, sha)
+	if err == operrors.ErrRetriable {
```

Comment:
It's safer to use `errors.Is(err, operrors.ErrRetriable)`. In case `err` is wrapped with some other error, it will unwrap all errors in the chain and compare them to `ErrRetriable`.